### PR TITLE
fix: clean code

### PR DESCRIPTION
**Pull request by notfilippo**
General code spring cleaning.  🧹 
Code should work as intended but further review is required before merge.

### CODE REVIEW

**Review by SteelAlloy — APPROVED** [review]
LGTM! 
Will merge it on monday (hopefully)

File: mod.ts [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+import { install } from "./src/commands/install.ts";
+import { upgrade } from "./src/commands/upgrade.ts";
 
-commandHandler();
+import { version } from "./src/version.ts";
+
+await new Command()
+  .name("eggs")
+  .version(version)
+  .description("nest.land - A package registry for Deno, on the permaweb")
```
Comment (t8) [inline mod.ts]:
```suggestion
  .description("nest.land - A module registry and CDN for Deno, on the permaweb")
```

File: src/commands/init.ts [hunk 2]
```diff
@@ -1,53 +1,56 @@
...
-            const content = decoder.decode(await Deno.readFile(`egg.${previousConfigFormat}`));
-            if(["yaml", "yml"].includes(previousConfigFormat)) {
-              let yamlConfig = parse(content);
-              // @ts-ignore
-              previousConfig = typeof yamlConfig == "object" ? yamlConfig : {};
-            }
-            else previousConfig = JSON.parse(content);
-        };
+  .version(version)
+  .description("Initiates a new package for the nest.land registry.")
```
Comment (t8) [inline src/commands/init.ts]:
```suggestion
  .description("Initiates a new module for the nest.land registry.")
```

File: src/global_module.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { readJson, writeJson } from "../deps.ts";
+
+export interface GlobalModule {
+  registry: string;
+  moduleName: string // TODO(@qu4k): ugly
```
Comment (t8) [inline src/global_module.ts:5]: Is this meant to be completed in this PR?

File: src/version.ts [hunk 4]
```diff
@@ -1 +1 @@
-export const version = "0.1.8";
+export const version = "0.1.8"; // TODO(@qu4k): ugly
```
Comment (t8) [inline src/version.ts:1]:
```suggestion
export const version = "0.1.9"; // TODO(@qu4k): ugly
```

We need to increment this version before the PR is merged

File: tests/commands/commands.ts [hunk 5]
```diff
@@ -1,5 +1,5 @@
 // The `init` command has not been included in the test as it has to be done maunally.
```
Comment (t8) [inline tests/commands/commands.ts]:
```suggestion
// The `init` command has not been included in the test as it has to be done manually.
```

**Review by t8 — CHANGES_REQUESTED** [review]
All commands have been tested manually. Everything works! Just address the things I mentioned, and this is good to merge!

File: src/global_module.ts (hunk 3 above)
Comment (SteelAlloy) [reply src/global_module.ts:5]: Later, it needs to be changed in the update handler as well if you want to change it @qu4k

File: src/version.ts (hunk 4 above)
Comment (notfilippo) [reply src/version.ts:1]: I think we should not, as a proper release can happen after we do the complete rewrite; we should keep what works (0.1.8) for now...

File: src/global_module.ts (hunk 3 above)
Comment (notfilippo) [reply src/global_module.ts:5]: part of the code cleaning, put in place to see if this style fits

File: src/version.ts (hunk 4 above)
Comment (t8) [reply src/version.ts:1]: Fair enough!

**Review by t8 — APPROVED** [review]
LGTM; thanks @qu4k!

**Pull request merged**